cmd/object-api-utils.go

```diff
@@ -239,10 +239,19 @@ func getCompleteMultipartMD5(parts []CompletePart) string {
 func cleanMetadata(metadata map[string]string) map[string]string {
 	// Remove STANDARD StorageClass
 	metadata = removeStandardStorageClass(metadata)
+	// Remove Tags
+	metadata = removeTags(metadata)
 	// Clean meta etag keys 'md5Sum', 'etag', "expires".
 	return cleanMetadataKeys(metadata, "md5Sum", "etag", "expires")
 }
 
+// removeTags deletes the tag entry (x-amz-tagging)
+// from the given map and returns the map
+func removeTags(metadata map[string]string) map[string]string {
+	delete(metadata, xhttp.AmzObjectTagging)
```

Comment:
@nitisht  there is no reason to return metadata its a memory can be done in one line - you don't need this function.